configs/strong_baselines/mask_rcnn_r50_fpn_syncbn-all_lsj_100e_coco.py

```diff
@@ -0,0 +1,21 @@
+_base_ = [
+    '../_base_/models/mask_rcnn_r50_fpn.py',
+    '../common/lsj_100e_coco_instance.py'
+]
+
+norm_cfg = dict(type='SyncBN', requires_grad=True)
+head_norm_cfg = dict(type='NaiveSyncBN', stats_mode='N', requires_grad=True)
+model = dict(
+    pretrained=None,
+    backbone=dict(frozen_stages=-1, norm_eval=False, norm_cfg=norm_cfg),
+    neck=dict(norm_cfg=norm_cfg),
```

Comment:
No need for neck because there is no BN.

Reply:
this will make FPN use syncBN, which is necessary